src/handlers/MemberLimitBtn.ts

```
@@ -32,12 +33,13 @@ export class MemberLimitButton extends ButtonHandler<[typeof MemberLimitId]> {
 @ModalRoute(MemberLimitModalId)
 export class MemberLimitModal extends ModalHandler<[typeof MemberLimitModalId]> {
     public async execute(): Promise<void> {
-        await this.defer()
+        const [server] = await Promise.all([database.findServer(this.event.guildId), this.defer()])
+        const t = getT(server?.language)
         const channel = this.event.channel
         const limit = parseInt(this.event.fields.getTextInputValue("limit").trim(), 10)
```

ESLint

[error] Move this declaration after the early exit.

(unicorn/no-declarations-before-early-exit)